SmartTutor/Startup.cs

```diff
@@ -38,5 +40,11 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
                 endpoints.MapControllers();
             });
         }
+
+        private void ActivateSmartTutorConsumer()
+        {
+            // TODO: Check is this a good place for this consumer
+            var consumer = new SmartTutorMessageConsumer();
```

Comment:
Who maintains a reference to this object and protects it from being garbage collected?
**We should research how consumers are handled in Startup.cs**

Reply:
[solution](https://github.com/Clean-CaDET/platform/pull/10#issuecomment-800631425)